helpers/html-elem-context-provider.js

```diff
@@ -0,0 +1,32 @@
+const attributeCallbacks = new Map();
+
+const contextObserver = new MutationObserver(mutations => {
+	mutations.forEach(mutation => {
+		const callbacks = attributeCallbacks.get(mutation.attributeName);
+		if (callbacks !== undefined && callbacks.length > 0) {
+			callbacks.forEach(callback => callback());
+		}
```

Comment:
Nit: could probably be:

```suggestion
		attributeCallbacks.get(mutation.attributeName)?.forEach(callback => callback());
```

Reply:
I'm never going to get used to being able to use chaining. 😆